tests/test_smoke.py

```diff
@@ -168,8 +185,10 @@
 # Managed job functions
 
 _WAIT_UNTIL_MANAGED_JOB_STATUS_CONTAINS_MATCHING_JOB_NAME = _WAIT_UNTIL_JOB_STATUS_CONTAINS_MATCHING_JOB_NAME.replace(
-    'sky queue {cluster_name}',
-    'sky jobs queue').replace('awk "\\$2 == ', 'awk "\\$3 == ')
+    'sky queue {cluster_name}', 'sky jobs queue').replace(
+        'awk "\\$2 == \\"{job_name}\\"',
+        'awk "\\$2 == \\"{job_name}\\" || \\$3 == \\"{job_name}\\"').replace(
+            _ALL_JOB_STATUSES, _ALL_MANAGED_JOB_STATUSES)
```

Comment:
While this reuse is good, it's getting harder to tell what args/fields need to be set in each string.format when using these string.

For example, I cannot tell from a simple glance here what fields need to be set when calling `_WAIT_UNTIL_MANAGED_JOB_STATUS_CONTAINS_MATCHING_JOB_NAME.format`. 

Can we instead of make these strings into methods that have well defined args? Something like:
```
def _WAIT_UNTIL_MANAGED_JOB_STATUS_CONTAINS_MATCHING_JOB_NAME(job_name, ...):
  cmd = _WAIT_UNTIL_JOB_STATUS_CONTAINS_MATCHING_JOB_NAME(...)
  ...
  return cmd.replace('sky queue {cluster_name}', 'sky jobs queue')...
```

That will make these wait commands much friendlier to use :)

Reply:
Sure